src/backend/looped/self/off_radar.cpp

```diff
@@ -12,15 +13,20 @@ namespace big
 
 		virtual void on_tick() override
 		{
-			scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = true;
-			*scr_globals::offradar_time.at(57).as<int*>() = NETWORK::GET_NETWORK_TIME() + 1;
+			if (g.self.ghost_org)
+				MISC::SET_BIT(script_global(2794162).at(4667).as<int*>(), 2);
+			*scr_globals::globalplayer_bd.at(PLAYER::PLAYER_ID(), 463).at(210).as<bool*>() = true;
+			*script_global(2672505).at(57).as<int*>() = NETWORK::GET_NETWORK_TIME() + 1;
 		}
 
 		virtual void on_disable() override
 		{
-			scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = false;
+			if (!g.self.ghost_org)
+				MISC::CLEAR_BIT(script_global(2794162).at(4667).as<int*>(), 2);
+			*scr_globals::globalplayer_bd.at(PLAYER::PLAYER_ID(), 463).at(210).as<bool*>() = false;
 		}
 	};
 
 	off_radar g_off_radar("otr", "OFF_RADAR", "OFF_RADAR_DESC", g.self.off_radar);
+	bool_command ghost_org("ghostorg", "Ghost Org", "Ghost Organization for free.", g.self.ghost_org);
```

Comment:
No need to add "for free", perhaps write "Use Ghost Organisation instead of standard off radar."

I'm open for other suggestions.

Reply:
I also realized i forgot to add it to the self view lmao